buildscripts/update-lobster.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# this script pulls the evergreen-ci/lobster repo and copies the files so that
+# they can be served by evergreen
+# it should only be run from the base evergreen directory
```

Comment:
Instead of putting a comment that it should be run from the evergreen directory, check that EVGHOME is set before continuing.